lms/templates/ccx/schedule.html

```diff
@@ -29,22 +29,24 @@
   <div id="new-ccx-schedule"></div>
 </div>
 
-<section id="enter-date-modal" class="modal" aria-hidden="true">
-  <div class="inner-wrapper" role="dialog">
+<section id="enter-date-modal" class="modal"
+         tabindex="-1" role="dialog" aria-labelledby="ccx_schedule_set_date_heading">
+  <div class="inner-wrapper">
     <button class="close-modal">
-      <i class="icon fa fa-remove"></i>
+      <i class="icon fa fa-remove" aria-hidden="true"></i>
       <span class="sr">
         ${_("Close")}
       </span>
     </button>
     <header>
-      <h2></h2>
+      <h2 id="ccx_schedule_set_date_heading"></h2>
     </header>
     <form role="form">
       <div class="field datepair">
-        <label></label>
-        <input placeholder="Date" class="date" type="text" name="date"/ size="11">
-        <input placeholder="Time" class="time" type="text" name="time"/ size="6">
+        <input placeholder="Date" class="date" type="text" name="date"
+               id="date" size="11" aria-label="${_('four digit year dash two digit month dash to digit day')}">
```

Comment:
A string like

_('format four digit year dash two digit month dash two digit day')

should have a translators comment because out of context this makes no sense. Perhaps, "Translators: This explains to people using a screen reader how to interpret the format of YYYY-MM-DD"

Also typo (to should be two)